magpie/adapter/__init__.py

```diff
@@ -32,7 +31,7 @@ def debug_cookie_identify(request):
         - :class:`pyramid.authentication.AuthTktCookieHelper`
         - :class:`pyramid.authentication.AuthTktAuthenticationPolicy`
     """
-    cookie_inst = request._get_authentication_policy().cookie
+    cookie_inst = request._get_authentication_policy().cookie  # noqa: W0212
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Access to a protected member _get_authentication_policy of a client class](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)